config/github/src/main/kotlin/GitHubConfigFileProvider.kt

```diff
@@ -93,6 +101,19 @@ class GitHubConfigFileProvider(
          */
         val TOKEN = Path("gitHubApiToken")
 
+        /**
+         * Configuration property that defines the root path of a file-based cache. If this is defined, the provider
+         * uses a [GitHubConfigFileCache] object to cache the fetched configuration files. Otherwise, caching is
+         * disabled.
+         */
+        const val CACHE_DIRECTORY = "gitHubCacheDirectory"
+
+        /**
+         * Configuration property that defines the interval for obtaining read locks (in seconds) when using the
+         * file-based configuration cache. This is only evaluated if a cache directory is defined.
+         */
+        const val LOCK_CHECK_INTERVAL = "gitHubCacheLockCheckInterval"
```

Comment:
In would be nice to have the unit in the name of the property.

Reply:
Added the suffix "Sec" to the property name.

Reply:
Could you add it to the Kotlin property name too ?

Reply:
Sure, but is this really needed? The unit is mentioned in the comment and is also obvious when the constant is used.

Reply:
I read somewhere is best practice.
But you can leave it if you want.

Reply:
I changed it - if it is best practice.